### ISSUE TITLE
Java 11 warning for protobuf illegal reflective access

### ISSUE THREAD

**Issue by antoniy** [signalfx/signalfx-java#62]
We've doing a migration of our application to Java 11 and Spring Boot 2.1.0 and we get these warning messages in our logs:

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.github.os72.protobuf351.UnsafeUtil (file:/Users/user/.m2/repository/com/signalfx/public/signalfx-java/0.0.48/signalfx-java-0.0.48.jar) to field java.nio.Buffer.address
WARNING: Please consider reporting this to the maintainers of com.github.os72.protobuf351.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

Not sure if this repository is the right one to reise the issue but I considered that we are using signalfx library and not protobuf so here should be the best place to report the issue.

**Comment by mpetazzoni** [comment]
I think that would be something to take up with the os72 shaded protobuf libraries that we use (https://github.com/os72) ?

We haven't looked at Java 11 support yet for `signalfx-java`.

**Comment by finci-square** [comment]
+1 I'm using this in java 11

**Comment by MatthewRyanRead** [comment]
@mpetazzoni Protobuf fixed this in 3.7.0 according to https://github.com/protocolbuffers/protobuf/issues/5549, and there is a shaded version of 3.11.1 at https://github.com/os72/protobuf-java-shaded-3-11-1.  So I think it's just signalfx-java that needs to bump those versions.

**Comment by jessicadeery** [comment]
+1 for this.
